### ISSUE TITLE
Draft User Guide

### ISSUE THREAD

**Issue by ccjernigan** [zcash/secant-android-wallet#406]

**Comment by geffenz** [comment]
I am collecting questions and user-guide topics now. I'll be compiling this document here: https://docs.google.com/document/d/1tjh3ZRV9EcRCM9NCfCxYVRlH0I9yvHes67LfiE1sWMs/edit?usp=sharing

**Comment by geffenz** [comment]
This sprint - Will create the actual URL so the app can hyperlink it (and validate with infra)

**Comment by geffenz** [comment]
Location is set for https://electriccoin.co/wallet-resources/user-guide/

Content is waiting on actual user feedback and design reviews. We can duplicate some wallet use recommendations too.

**Comment by geffenz** [comment]
We will be collaborating with the (once hired) community manager on this as well.

**Comment by geffenz** [comment]
This is on hold until we have a build to work off of, and are past the current zcash sync issues.
